### PR TITLE
VeraCrypt: update to 1.25.7

### PR DESCRIPTION
**Pull request by gbeliako**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-LIBC)
 - I built this PR locally for these architectures (if supported. mark crossbuilds):
 - Crossbuilds: 
  - aarch64-musl
  - armv7l

### CODE REVIEW

File: srcpkgs/VeraCrypt/template [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Template file for 'VeraCrypt'
 pkgname=VeraCrypt
-version=1.24
+version=1.25.7
 revision=1
-_realversion="${version}-Update2"
-wrksrc="${pkgname}-${pkgname}_${_realversion}"
+#_realversion="${version}-Update2"
```
Comment (ericonr) [inline srcpkgs/VeraCrypt/template:5]: You can simply remove this line.

Comment (gbeliako) [reply srcpkgs/VeraCrypt/template]: Done. Thanks

**Comment by paper42** [comment]
could you please add the email address used in the commit to your github profile?

**Comment by gbeliako** [comment]
done

**Pull request merged**